build/pom.xml

```diff
@@ -251,8 +251,15 @@
         <version>${tycho-version}</version>
         <configuration>
           <argLine>-ea</argLine>
-          <parallel>classes</parallel>
-          <threadCount>4</threadCount>
+          <!-- There seems to be an issue with executing several tests in parallel in that
+               it can happen that failed tests are eventually reported as having succeeded.
+               A resolution is to set forkCount=1 and reuseForks=false
+               See, e.g. the following stackoverflow post:
+               https://stackoverflow.com/questions/3365628/junit-tests-pass-in-eclipse-but-fail-in-maven-surefire -->
+          <!-- parallel>classes</parallel -->
+          <!-- threadCount>4</threadCount -->
+          <reuseForks>false</reuseForks>
+          <forkCount>1</forkCount>
```

Comment:
I understand that this will effectively prevent our tests from being executed in parallel. Is that correct?

Reply:
Yes. Unfortunately I didn't find another solution.